shield_test.go

```diff
@@ -6,12 +6,14 @@ import (
 	"testing"
 )
 
+var newShield = Default
```

Comment:
Why not assigning the return value of `Default` instead? Or better yet, using the `DefaultShield` variable? This will also save you of having to call `newShield()` on each test function.